src/HTMLImage.js

```diff
@@ -106,7 +106,7 @@ export default class HTMLImage extends PureComponent {
         return (
             <Image
               source={source}
-              style={[style, { width: this.state.width, height: this.state.height, resizeMode: 'cover' }]}
+              style={[{ width: this.state.width, height: this.state.height, resizeMode: 'cover' }, style]}
```

Comment:
I do understand the value of overriding `resizeMode`, but I am skeptical about the width and height, because there is now way to know in advance if it will fit the device dimensions. Could you explain the rationale?